utils.go

```diff
@@ -288,3 +288,8 @@ func calcMapHash(m map[string]string) (uint32, error) {
 	}
 	return h.Sum32(), nil
 }
+
+func isToCache(length int64, s *scope) bool {
+	maxPayloadSize := s.user.cache.MaxPayloadSize
+	return length <= maxPayloadSize
```

Comment:
open question: should we had some metrics in order to know the number of queries not cached because they're too big